apps/dev-portal-web/lang/en.json

```diff
@@ -182,5 +182,21 @@
   "apps.b2c.sections.publishing.publishForm.items.info.label": "Information about the application",
   "apps.b2c.sections.publishing.publishForm.items.build.label": "Application build",
   "apps.b2c.sections.publishing.publishForm.items.build.select.placeholder": "Select build",
-  "apps.b2c.sections.publishing.publishForm.actions.publish": "Publish"
-}
\ No newline at end of file
+  "apps.b2c.sections.publishing.publishForm.actions.publish": "Publish",
+
+  "docs.widgets.amountDistributionCalculator.paymentSettings": "Payment settings",
```

Comment:
Should we keep it flat or add depth as in other translations?

`docs.widgets.amountDistributionCalculator.paymentAmount` -> `docs.widgets.amountDistributionCalculator.recipentCard.paymentAmount`

Reply:
I think `docs.widgets.amountDistributionCalculator` is enough for this small component.

Reply:
The goal of translation keys is be as transparent as possible. From this string I can get, that this strings are used somewhere in widget, but no I have no idea where and for what. Is it label / placeholder / text...I've added some example in another comment